<launch>
    <param name="planner_cmd_topic" value="/planning/pos_cmd" type="str"/>
    <!-- 起飞后发一个目标点给飞控 -->
    <node pkg="mission_pkg" type="bt_ros" name="fly2point" output="screen">
      <param name="config_path" value="$(find mission_pkg)/config/single_planner_rviz.xml" type="str"/>
    </node>
</launch>
